sklearn/decomposition/_dict_learning.py

```diff
@@ -2412,7 +2412,7 @@ def fit(self, X, y=None):
             self.n_steps_ = i + 1
             self.n_iter_ = np.ceil(self.n_steps_ / n_steps_per_iter)
         else:
-            # TODO remove this branch in 1.3
+            # TODO remove this branch in 1.4
```

Comment:
In 1.2, the deprecation was postponed to 1.4 but the comment was not modified accordingly.